huggle/mainwindow.ui

```diff
@@ -81,16 +81,16 @@
      <x>0</x>
```

Comment:
Which version of QtCreator you have? It basically completely rewrote the ui file...

Reply:
I've updated my qtcreator, it looks ok now